buildSrc/src/main/groovy/org/grails/documentation/DocumentationPage.groovy

```diff
@@ -73,26 +74,23 @@ class DocumentationPage {
         Collection<DocumentationCategory> categories = categories(modules)
         html.div(class: 'content') {
             html.div(class: "twocolumns") {
-                if (preRelease > latest) {
-                    html.div(class: "odd column") {
+                html.div(class: "odd column"){
+                    if (preRelease > latest) {
                         mkp.yieldUnescaped(renderDocumentation(preRelease.versionText))
-                    }
-                } else {
-                    html.div(class: "odd column") {
                         mkp.yieldUnescaped(renderDocumentation('snapshot'))
```

Comment:
Is this correct here... that you want to render 'snapshot' here on line 80 *and* on line 83?

If that's not correct, let's fix the name(s).

If it is correct, I think the following conditional would be clearer (since you're doing snapshot in both cases):
```
if (preRelease > latest) {
    mkp.yieldUnescaped(renderDocumentation(preRelease.versionText)
}
mkp.yieldUnescaped(renderDocumentation('snapshot'))

Reply:
Line numbers referring to the proposed change, not the original code.

Reply:
Ah yeah, a bit of logic I should have cleaned up